src/components/GameCard.tsx

```diff
@@ -30,39 +32,53 @@ export const GameCard: React.FC<GameCardProps> = ({
   }, [game.sequelFamily]);
   const upcoming = useMemo(() => {
     if (game.releaseDate) {
-      return game.releaseDate === "TBA"
-        ? true
-        : new Date(game.releaseDate) > new Date();
-    } else {
-      return false;
+      return (
+        game.releaseDate === "TBA" || new Date(game.releaseDate) > new Date()
+      );
+    }
+
+    if (game.year) {
+      return new Date(game.year) > new Date();
     }
-  }, [game.releaseDate]);
+
+    return false;
+  }, [game.releaseDate, game.year]);
 
   return (
-    <Card className="dark:bg-zinc-900">
-      <CardHeader className="pb-2">
-        {upcoming && (
+    <Card className="dark:bg-zinc-900 h-full flex flex-col">
+      <CardHeader className="pb-2 flex-none">
+        {upcoming ? (
           <div>
             <Badge variant="destructive" className="text-base">
               Not Released
             </Badge>
           </div>
+        ) : (
+          <div>
+            <Badge variant="released" className="text-base">
+              Released
+            </Badge>
+          </div>
         )}
-        {game.releaseDate && (
-          <p className="text-sm mt-1">
-            <span className="font-light">Release Date:</span>{" "}
-            <span className="font-normal">{game.releaseDate}</span>
-          </p>
-        )}
+        <p className="text-sm mt-1">
+          <span className="font-light">Release Date:</span>{" "}
+          <span className="font-normal">
+            {game.releaseDate || "Not available"}
+          </span>
+        </p>
```

Comment:
The releaseDate is not loaded on items, so most show like this:

<img width="299" alt="Screenshot 2024-12-19 at 09 38 26" src="https://github.com/user-attachments/assets/7cd0d24e-d29a-4201-b4fb-abd6cc59059d" />